Traceback (most recent call last):
  File "/g/data/xf3/zz3507/envs/syri_1.4/syri/syri/bin/syri", line 165, in <module>
    from syri.pyxFiles.synsearchFunctions import readCoords
  File "syri/pyxFiles/synsearchFunctions.pyx", line 9, in init syri.pyxFiles.synsearchFunctions
  File "/g/data/xf3/miniconda/envs/syri_1.4/lib/python3.5/site-packages/igraph/__init__.py", line 34, in <module>
    from igraph._igraph import *
ImportError: libiconv.so.2: cannot open shared object file: No such file or directory
